Project.toml

```
@@ -1,6 +1,6 @@
 name = "SpecialFunctions"
 uuid = "276daf66-3868-5448-9aa4-cd146d93841b"
-version = "2.6.1"
+version = "2.6.2"
```

IMO a Julia implementation of `erf` is a feature, not a bug fix, so the new version should be

```suggestion
version = "2.7.0"
```